skptesting/simple-opa-bundle/allow.rego

```diff
@@ -0,0 +1,11 @@
+package envoy.authz
+
+import rego.v1
+
+default allow := false
+
+# METADATA
+# entrypoint: true
```

Comment:
nitpick: clean comments if not used

Reply:
This is actually an [annotation](https://www.openpolicyagent.org/docs/latest/policy-language/#annotations) added to the policy.